test/user.e2e-spec.ts

```diff
@@ -206,7 +206,7 @@ describe('User e2e', () => {
   });
 
   // LIST USERS
-  it.skip('list view of users', async () => {
+  it.only('list view of users', async () => {
```

Comment:
```suggestion
  it('list view of users', async () => {
```

Reply:
sorry about that, been a distracting day.

I also added the no-only-tests linter